packages/esm-ward-app/src/ward-workspace/patient-transfer-bed-swap/patient-transfer-request-form.component.tsx

```diff
@@ -229,7 +229,7 @@ export default function PatientTransferForm({
         <Button
           type="submit"
           size="xl"
-          disabled={isLoadingEmrConfiguration || isSubmitting || errorFetchingEmrConfiguration}>
+          disabled={isLoadingEmrConfiguration || isSubmitting || errorFetchingEmrConfiguration || !patient}>
```

Comment:
ah... is it just that we are waiting for the patient to load?  In that case, is just adding the "!patient" flag here enough?

Reply:
Yeah, the load meanwhile takes a few seconds, i.e waiting to read it from the store.